ci/test_conda.sh

```diff
@@ -9,23 +9,29 @@ set -euo pipefail
 rapids-logger "Starting Conda Package Test"
 
 rapids-logger "Creating Test Environment"
-rapids-mamba-retry create -n test
+# TODO: replace this with rapids-dependency-manager
+rapids-mamba-retry create -n test \
+  click \
+  pytest \
+  clangdev >=18 \
+  cuda-nvcc >=12.5 \
+  cuda-version >=12.5 \
+  cuda-nvrtc \
+  numba >=0.59 \
+  numba-cuda >=0.2.0 \
+  pynvjitlink >=0.2 \
+  cuda-cudart-dev \
+  python=${RAPIDS_PY_VERSION}
 
 # Temporarily allow unbound variables for conda activation.
 set +u
 conda activate test
 set -u
 
-rapids-logger "Add conda build output dir to channel"
-
-conda index  $RAPIDS_CONDA_BLD_OUTPUT_DIR/
-conda config --add channels $RAPIDS_CONDA_BLD_OUTPUT_DIR
-
 rapids-print-env
 
 rapids-mamba-retry install \
```

Comment:
Try to fuse this into the `rapids-mamba-retry create -n test` command. If you can do a single conda solve, your CI will be faster and more correct.

https://github.com/rapidsai/build-planning/issues/22

Reply:
I had thought that when we installed all dependencies for the libs first, then force installing from `pwd/conda-repo` channel, it should've installed from the local channel. Apparently that's not the case.